### PR TITLE
feature/team-logo

### PR DESCRIPTION
**Pull request by Kachyr**
The component displays the team logo and takes three properties:
name - name of the team
iconUrl - url with icon image 
isUnchecked - boolean property which changing styles of the component

### CODE REVIEW

File: src/app/shared/team-radio-switch/team-radio-switch.component.html [hunk 1]
```diff
@@ -7,8 +7,11 @@
       [(ngModel)]="valueId"
       #refInput1
     />
-    <img [src]="team1.iconUrl" [alt]="team1.name" />
-    <div class="input-name">{{ team1.name }}</div>
+    <app-team-logo
+      [name]="team1.name"
+      [iconUrl]="team1.iconUrl"
+      [isUnchecked]="refInput2.checked"
```
Comment (Yurii-88) [inline src/app/shared/team-radio-switch/team-radio-switch.component.html:13]: There is semantic inconsistency in a bound property and value you assign to it

File: src/app/shared/team-logo/team-logo.component.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.logo-body {
+  display: flex;
+  flex-direction: row;
```
[the remark this answers is not in the document]
Comment (Kachyr) [reply src/app/shared/team-logo/team-logo.component.css:3]: done

File: src/app/shared/team-radio-switch/team-radio-switch.component.html (hunk 1 above)
Comment (Kachyr) [reply src/app/shared/team-radio-switch/team-radio-switch.component.html:13]: I don't know if I understood the comment correctly, but I changed `[isUnchecked]="refInput2.checked"` that I think should correct the semantic inconsistency

File: src/app/shared/team-radio-switch/team-radio-switch.component.html [hunk 3]
```diff
@@ -19,7 +22,10 @@
       [(ngModel)]="valueId"
       #refInput2
     />
-    <img [src]="team2.iconUrl" [alt]="team2.name" />
-    <div class="input-name">{{ team2.name }}</div>
+    <app-team-logo
+      [name]="team2.name"
+      [iconUrl]="team2.iconUrl"
+      [isUnchecked]="refInput1.checked"
```
[the remark this answers is not in the document]
Comment (Kachyr) [reply src/app/shared/team-radio-switch/team-radio-switch.component.html:28]: I don't know if I understood the comment correctly, but I changed [isUnchecked]="refInput1.checked" that I think should correct the semantic inconsistency

**Pull request merged**